Doc/howto/perf_profiling.rst

```diff
@@ -147,19 +147,27 @@ Instead, if we run the same experiment with perf support activated we get:
 
 
 
-Enabling perf profiling mode
-----------------------------
+How to enable the ``perf`` profiling mode
+-----------------------------------------
 
-There are two main ways to activate the perf profiling mode. If you want it to be
-active since the start of the Python interpreter, you can use the `-Xperf` option:
+There are three ways to activate the ``perf`` profiling mode:
+using the :option:`-Xperf <-X>` command-line option,
```

Comment:
We could maybe make these bullet points?

Reply:
Yes, that's nice. Added in 31c0eb819c2b951051e5cb12d497db01af28bcd4.